examples/asa/assets/5-contract-owned-asa-approval.py

```diff
@@ -0,0 +1,80 @@
+from pyteal import *
```

Comment:
Insteal of using *contract-owned-asa* in file name, can we just use *contract-asa* ?

Reply:
yes